portal-api/src/v1/users/validation/rules/select-at-least-one-role.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Validates that if the roles are present at least one is not null
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneRole = (user, change) => {
+  if (!change.roles || (Array.isArray(change.roles) && change.roles.every((item) => item === null)) || change.roles.length === 0) {
```

Comment:
Overly complicated statement still?

Reply:
Simplified to `if (!(change.roles && change.roles.some((item) => item)))`